src/sentry/event_manager.py

```diff
@@ -451,6 +464,12 @@ def save(self, project, raw=False):
             'first_seen': date,
             'time_spent_total': time_spent or 0,
             'time_spent_count': time_spent and 1 or 0,
+            'data': {
+                'type': event_type.key,
+                # we cache the events metadata on the group to ensure its
+                # accessible in the stream
+                'metadata': event_type.get_metadata(),
```

Comment:
the issue needs this cached, but events don't

I imagine what'd happen in the UI is the stream somehow gets the metadata cache here, but individual events compute it on the fly. That's a bit odd in case it changes, but otherwise we need to add some (duplicated) storage to the Event.data packet which also stores this.